pkg/repository/influx.go

```diff
@@ -0,0 +1,84 @@
+package repository
+
+import (
+	"context"
+
+	client "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api"
+	"github.com/influxdata/influxdb-client-go/v2/log"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/credentials"
+
+	"github.com/instill-ai/pipeline-backend/config"
+	"github.com/instill-ai/pipeline-backend/pkg/logger"
+)
+
+// InfluxDB reads and writes time series data from InfluxDB.
+type InfluxDB struct {
+	client client.Client
+	api    api.WriteAPI
+}
+
+// NewInfluxDB returns an initialized InfluxDB repository.
+func NewInfluxDB(ctx context.Context) (*InfluxDB, error) {
```

Comment:
@donch1989 let me know what you think about this. In [`mgmt-backend`](https://github.com/instill-ai/mgmt-backend/blob/main/pkg/repository/influx.go) InfluxDB is part of the `repository` package, so I think it's good to follow that structure here as well.

Reply:
Can do